gooddata-sdk/gooddata_sdk/catalog/organization/entity_model/llm_endpoint.py

```diff
@@ -0,0 +1,239 @@
+            token=None,  # Token is not returned for security reasons
+            description=safeget(ea, ["description"]),
+            provider=safeget(ea, ["provider"]),
+            base_url=safeget(ea, ["baseUrl"]),
+            llm_organization=safeget(ea, ["llmOrganization"]),
+            llm_model=safeget(ea, ["llmModel"]),
+            workspace_ids=safeget(ea, ["workspaceIds"]),
+        )
+        return cls(
+            id=entity["id"],
+            attributes=attr,
+        )
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class CatalogLlmEndpointPatch(Base):
+    id: str
+    attributes: Optional[CatalogLlmEndpointPatchAttributes] = None
+
+    @staticmethod
+    def client_class() -> type[JsonApiLlmEndpointPatch]:
+        return JsonApiLlmEndpointPatch
+
+    @classmethod
+    def init(
+        cls,
+        id: str,
+        title: Optional[str] = None,
+        token: Optional[str] = None,
+        description: Optional[str] = None,
+        provider: Optional[str] = None,
+        base_url: Optional[str] = None,
+        llm_organization: Optional[str] = None,
+        llm_model: Optional[str] = None,
+        workspace_ids: Optional[list[str]] = None,
+    ) -> CatalogLlmEndpointPatch:
+        return cls(
+            id=id,
+            attributes=CatalogLlmEndpointPatchAttributes(
+                title=title,
+                token=token,
+                description=description,
+                provider=provider,
+                base_url=base_url,
+                llm_organization=llm_organization,
+                llm_model=llm_model,
+                workspace_ids=workspace_ids,
+            ),
+        )
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class CatalogLlmEndpointAttributes(Base):
+    title: str
+    token: str
+    description: Optional[str] = None
+    provider: Optional[str] = None
+    base_url: Optional[str] = None
+    llm_organization: Optional[str] = None
+    llm_model: Optional[str] = None
+    workspace_ids: Optional[list[str]] = None
+
+    @staticmethod
+    def client_class() -> type[JsonApiLlmEndpointInAttributes]:
+        return JsonApiLlmEndpointInAttributes
+
+    def to_api(self) -> dict:
+        """Convert to API format (camelCase). Only include non-None attributes."""
+        result = {
+            "title": self.title,
+            "token": self.token,
+        }
+        if self.description is not None:
+            result["description"] = self.description
+        if self.provider is not None:
+            result["provider"] = self.provider
+        if self.base_url is not None:
+            result["baseUrl"] = self.base_url
+        if self.llm_organization is not None:
+            result["llmOrganization"] = self.llm_organization
+        if self.llm_model is not None:
+            result["llmModel"] = self.llm_model
+        if self.workspace_ids is not None:
+            result["workspaceIds"] = self.workspace_ids
+        return result
```

Comment:
Is it necessary to do this? `Base` class has an implementation for `to_api` which works when `client_class` is specified.